sources/editor/Stride.GameStudio/app.manifest

```diff
@@ -56,7 +56,6 @@
   </application>
   
   <!-- Enable themes for Windows common controls and dialogs (Windows XP and later) -->
-  <!--
```

Comment:
What are the implications here? Does it even impact WPF, I thought it was only for C++ applications or WinForms applications, as they use ComPlus components.

This could affect the WinForms control that we use to render the view in the editor, which I'm not sure we should tinker with.

Reply:
I enabled these due to Win32 buttons being visible native message boxes; as stated in the commit "Also use Windows visual styles; system message boxes look better then (one appears when declining the privacy policy window)."
I did not notice any differences in the behavior of the editor view, but it is probably better to leave it out then because the editor view is more important.

Reply:
Ok. I did not think we still had some native windows.